streaming/nifi/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "configure_kubectl" {
+  description = "Configure kubectl: make sure you're logged in with the correct AWS profile and run the following command to update your kubeconfig"
+  value       = module.eks_blueprints.configure_kubectl
```

Comment:
This will need to be changed or removed as upstream does not have the kubectl output.